app/src/main/java/org/simple/clinic/cvdrisk/CVDRiskRange.kt

```diff
@@ -12,6 +13,14 @@ data class CVDRiskRange(
     val max: Int,
 ) : Parcelable {
 
+  @Transient
+  @IgnoredOnParcel
+  val level = CVDRiskLevel.compute(this)
+
+  @Transient
+  @IgnoredOnParcel
+  val canPrescribeStatin = max >= 10
```

Comment:
Instead of defining the variables directly, can we instead use the property getter like so?

```kotlin
  val canPrescribeStatin: Boolean
    get() = max >= 10
```

That way we don't have to deal with the serializer/parcelizer.